cdci_data_analysis/analysis/parameters.py

```diff
@@ -476,14 +487,8 @@ def __init__(self, value=None, T_format='isot', name=None, Time_format_name=None
 
         self._set_time(value, format=T_format)
 
-    def set_par(self, value, units=None):
-        # sets the value to the format initially specified in the form
-        if units is not None:
-            self.units = units
-        self.value = value
-        # return the relative isot format value
-        # afterwards also the units_name will be set to isot
-        return self._astropy_time.isot
+    def get_value_in_default_format(self, value) -> Union[str, float, None]:
+        return getattr(self._astropy_time, self.default_units)
```

Comment:
before we freese it, any change to rename it to `default_unit`? It's a single unit. As opposed to `allowed_units` where it's many.
It's a bit inconsistent throughout. But I guess at least `default_units` is not yet used in plugins?

Reply:
> before we freese it, any change to rename it to `default_unit`? It's a single unit.
yes I guess it's not gonna be a problem

> But I guess at least `default_units` is not yet used in plugins?
It has been introduced with this PR so, no, I will check anyway

Reply:
so, actually I realized that I named it as `default_units` to be consistent with `units`

Reply:
> so, actually I realized that I named it as default_units to be consistent with units

Yeah that's what I meant by `inconsistent throughout`. Meh. Maybe let's add a TODO instead? Else need to preserve interface carefully.

Reply:
indeed, I would stick with this (wrong) approach, I will add the TODO